packages/odyssey-react-mui/src/Link.tsx

```diff
@@ -27,7 +34,7 @@ export type LinkProps = {
   /**
    * The content within the Link
    */
-  children: React.ReactNode;
+  children?: ReactNode;
```

Comment:
When passing `children`, do we have it setup to require an `ariaLabel`?

Reply:
We do not. `children` can be anything here, or nothing now 😆 . `aria-label` is not usually required

Reply:
`aria-label` should be required if we have `children` passed. Now that it's properly optional, that's where we need to add a restriction that says if `children` is required, then `ariaLabel` is required. If `children` is optional, then we require `linkText`.

Reply:
That doesn't really solve the problem we were looking to solve with this change. When interpolating strings of text using i18n `<Trans>` component, the text for the link is not available. It gets passed in as children by `<Trans>`. This causes TS errors with our current implementation. 
```js
<Trans
  i18nKey="your.key.name.here"
  components={[
  <Link
    key={0}
    href="https://whatever.com"
    target="_blank"
    rel="noopener noreferrer"
   />,
  ]}
/>
```